### PR TITLE
feat/end block anchor

### PR DESCRIPTION
**Pull request by kevinslin**
- remove site banner
- Minor grammar edit: "2" -> "two".
- sync
- end positional ref

### CODE REVIEW

File: vault/dendron.topic.note-reference.md [hunk 1]
```diff
@@ -97,6 +97,18 @@ This reference will reference the start of a note until it encounters the first
...
 ![[dendron.topic.note-reference.sample#^begin]]
 
+#### End Positional Reference
+
+This reference will reference the end of a note. 
+
+- NOTE: the `end` positional reference cannot be used in the `start-anchor` position of a [[#range-reference]]
+
+```md
+![[dendron.topic.note-reference.sample#^end]]
```
Comment (jonathanyeung) [inline vault/dendron.topic.note-reference.md:107]: Would anyone ever want to do this? Should we change the example to use a range reference instead?

Comment (jonathanyeung) [reply vault/dendron.topic.note-reference.md:107]: Er - ok maybe I'm misunderstanding this.  This example is showing what we _can't_ do? Why don't we instead show an example of when someone would want to use the end positional reference?

**Review by jonathanyeung — CHANGES_REQUESTED** [review]
some comments above

File: vault/dendron.topic.note-reference.md (hunk 1 above)
Comment (kevinslin) [reply vault/dendron.topic.note-reference.md]: ah, wrote the wrong example. updated this.

**Review by jonathanyeung — APPROVED** [review]

**Pull request merged**